superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -149,8 +149,10 @@ class ChartContainer extends React.Component {
     this.props.actions.removeChartAlert();
   }
 
-  renderVis() {
-    visMap[this.props.viz_type](this.state.mockSlice).render();
+  renderVis(heightChange) {
+    if (this.props.chartStatus === 'loading' || heightChange) {
+      visMap[this.props.viz_type](this.state.mockSlice).render();
```

Comment:
since we have the resize timer in exploreViewContainer, control panel and chart container with everything wrapped inside them will resize at the same time when actual resize is finished, I think in this case we don't have to resize the viz anymore?